qiskit_experiments/base_experiment.py

```diff
@@ -348,7 +349,7 @@ def _add_job_metadata(self, experiment_data: ExperimentData, job: BaseJob, **run
             run_options: backend run options for the job.
         """
         metadata = {
-            "job_id": job.job_id(),
+            "job_id": job.job_id,
```

Comment:
@chriseclectic This one too, thanks!

Reply:
```suggestion
            "job_id": job.job_id()
```
`job_id` is a method, not property: https://github.com/Qiskit/qiskit-terra/blob/main/qiskit/providers/job.py#L62